app/src/main/java/com/framgia/forder/screen/searchpage/SearchContainerViewModel.java

```diff
@@ -87,4 +86,9 @@ public void onSearchAction(String keyWord) {
             }
         };
     }
+
+    @Bindable
+    public boolean getClear() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#BooleanGetMethodName